.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "OCaml earlybird (experimental)",
+            "type": "ocaml.earlybird",
+            "request": "launch",
+            "program": "${workspaceRoot}/_build/default/bin/ocamlformat/main.bc",
```

Comment:
It would be great if this called `dune build _build/default/bin/ocamlformat/main.bc` before every runs.